x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/view/host_isolation_exceptions_list.tsx

```diff
@@ -125,6 +123,15 @@ export const HostIsolationExceptionsList = () => {
     [navigateCallback]
   );
 
+  const handleCloseDeleteDialog = (forceRefresh: boolean = false) => {
```

Comment:
please memoize this with `useCallback()`

Reply:
there's no win on memorizing this with useCallback. It is a pattern I see commonly in many of our files of using useMemo and useCallback in places where if any, there is a worse performance by using it.

Even to argue that the rendered button will re-render because the callback is different is not worthy. react is fast (and made) to handle those cases, the performance win is so minimal compared to how polluted and more complicated the code looks with useCallbacks and useMemo.

The bottom line: react is very well optimized, we should not try to out-smart it. useCallback and useMemo are useful for specific cases where you want to memo expensive renders and expensive calculations. Micro optimizations like this are not worth in the big scheme.